bundles/org.openhab.binding.resol/src/main/java/org/openhab/binding/resol/handler/ResolThingHandler.java

```diff
@@ -287,13 +287,15 @@ protected void packetReceived(Specification spec, Language lang, Packet packet)
                         if (b != null) {
                             ResolBridgeHandler handler = (ResolBridgeHandler) b.getHandler();
                             String value;
+                            Locale loc;
                             if (handler != null) {
-                                value = pfv.formatTextValue(pfv.getPacketFieldSpec().getUnit(), handler.getLocale());
+                                loc = handler.getLocale();
                             } else {
-                                value = pfv.formatTextValue(pfv.getPacketFieldSpec().getUnit(), Locale.getDefault());
+                                loc = Locale.getDefault();
                             }
```

Comment:
Your comment above: "vbus library returns the value in default locale".
So why not just using there like above `Locale.getDefault()` ?

Reply:
By the way, you changed nothing there, so let's close my comment.